frontend/src/components/pages/CreateReview/CreateReviewPage.tsx

```diff
@@ -193,8 +197,10 @@ const CreateReview = (): React.ReactElement => {
                 key={i}
                 index={i}
                 book={book}
-                showModal={setShowDeleteBookModal}
-                setIndex={setDeleteBookIndex}
+                showDeleteBookModal={setShowDeleteBookModal}
+                setDeleteBookIndex={setDeleteBookIndex}
+                showBookModal={onOpenBookModal}
+                setCurrBook={setCurrBook}
               />
             ))}
 
```

Comment:
I know this is not part of your ticket, but are we good to remove the commented code in this file?

Reply:
Not sure, i'm concerned that some of the code could be used for tags, @susheelk?

Reply:
Resolved in call